Debian don't have libjpeg-turbo now, disable this warning or compile will fail.
--- a/gtk/channel-display-mjpeg.c
+++ b/gtk/channel-display-mjpeg.c
@@ -86,7 +86,6 @@
     else
         st->mjpeg_cinfo.out_color_space = JCS_EXT_BGRX;
 #else
-#warning "You should consider building with libjpeg-turbo"
     st->mjpeg_cinfo.out_color_space = JCS_RGB;
 #endif
 
